docs/integrate-new-apps/anpm-packaging.md

```diff
@@ -0,0 +1,448 @@
+
+- Update firebase.json to include deploy targets and bundling configurations `firebase.json`:
+
+``` javascript
+{
+  "hosting": [
+    {
+      "public": "dist",
+      "target": "production",
+      "ignore": ["firebase.json", "**/.*", "**/node_modules/**"],
+      "rewrites": [
+        {
+          "source": "**",
+          "destination": "/index.html"
+        }
+      ],
+      "headers": [
+        {
+          "source": "**/*",
+          "headers": [
+            {
+              "key": "Feature-Policy",
+              "value": "autoplay=*"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "public": "dist",
+      "target": "staging",
+      "ignore": ["firebase.json", "**/.*", "**/node_modules/**"],
+      "rewrites": [
+        {
+          "source": "**",
+          "destination": "/index.html"
+        }
+      ],
+      "headers": [
+        {
+          "source": "**/*",
+          "headers": [
+            {
+              "key": "Feature-Policy",
+              "value": "autoplay=*"
+            }
+          ]
+        }
+      ]
+    }
+  ]
+}
+```
+
+- Set up GitHub actions for PR link deployment, Cypress testing, staging deployment, and npm publishing.
+- Use one of the established apps, such as roar-swr, as a template. If you have properly set up staging and hosting targets for the app, then it should suffice to copy and paste the existing workflow files.
+- Add necessary secrets to the GitHub repository for npm auth token, Firebase service account, Cypress, and Sentry.
+
+
+## Create projects in platforms
+
+### Sentry Dashboard
+1. Enter the Sentry dashboard with your account
+2. Go to "Projects" and select "Create Project"
+3. Select Browser Javascript
+4. For the frequencies select "I'll create my own alerts later"
+5. Add the project name and the team "#roar"
+6. Once the project is created, go to the configurations of the project and copy the "Client Keys (DSN)"
+
+
+### Cypress Dashboard
+1. Enter the Cypress dashboard with your account
+2. Go to "Projects" and select "New Project"
+3. Add the project name and select "Private"
+4. For the tests create only "test-pr"
+
+
+## Sentry Integration
+- Install Sentry packages:
+
+``` javascript
+npm install @sentry/browser@latest
+npm install @sentry/integrations@latest
+npm install @sentry/wasm@latest
+npm install @sentry/rollup-plugin
+npm install @sentry/webpack-plugin
+
+```
+
+Once you run this, copy the secret keys for:
+
+- CYPRESS_RECORD_KEY
+- FIREBASE_SERVICE_ACCOUNT_GSE_ROAR_ASSESSMENT
+- NPM_TOKEN
+- SENTRY_AUTH_TOKEN
+- SENTRY_ORG
+- SENTRY_PROJECT
+
+
+**Sentry wizard for readable source maps:**
+
+1. Run the Sentry wizard for readable source maps:
+
+    ```bash
+    npx @sentry/wizard@latest -i sourcemaps
+    ```
+
+   For both webpack and rollup configurations.
+
+**index.js:**
+
+- Remove the "fromDashboard" if statement and always initialize Sentry.
+
+**serve.js:**
+
+- Remove `initSentry` from `serve.js`.
```

Comment:
```suggestion
```
This should already be done across all of the apps now, so I think we can remove this.